README.md

```diff
@@ -675,6 +675,34 @@ if (isChangeset(model)) {
 - [`ember-changeset-hofs`](https://github.com/nucleartide/ember-changeset-hofs) - Higher-order validation functions
 - [`ember-bootstrap-changeset-validations`](https://github.com/kaliber5/ember-bootstrap-changeset-validations) - Adds support for changeset validations to `ember-bootstrap`
 
+## Alternative Use Cases
+
+If you wish to use a changeset from a template without a component you can use the `with` helper:
```

Comment:
This usage is already (somewhat) suggested at https://github.com/poteto/ember-changeset#disabling-automatic-validation - I'd rather not bloat the readme. :(

Reply:
Is there a way to make that more explicit? I've read this README several times and still didn't glean that use case. Perhaps I just skipped over that because I wasn't thinking about validations which that section suggests is tied to `{{with}}`. I think the implied usage is confusing it would help if it wasn't tied to validations. Thoughts?

Reply:
Updated here: https://github.com/poteto/ember-changeset/tree/develop/nested-keys#usage